# patch to make rungms wrapper script for GAMESS work on our
# gentoo install

--- gamess/rungms	2005-07-23 20:33:54.000000000 +0000
+++ gamess-gentoo/rungms	2005-09-04 17:01:19.000000000 +0000
@@ -39,7 +39,7 @@
 #       its own list of further requirements.
 #
 set TARGET=sockets
-set SCR=/scr/$USER
+set SCR=./
 #
 set JOB=$1      # name of the input file xxx.inp, give only the xxx part
 set VERNO=$2    # revision number of the executable created by 'lked' step
@@ -90,11 +90,11 @@
 #  is therefore most suitable to be stored only once, on the file server.
 #
 set echo
-setenv ERICFMT ~mike/gamess/ericfmt.dat
+setenv ERICFMT /usr/share/gamess/ericfmt/ericfmt.dat
 setenv  EXTBAS /dev/null
-setenv IRCDATA ~/scr/$JOB.irc
+setenv IRCDATA $SCR/$JOB.irc
 setenv   INPUT $SCR/$JOB.F05
-setenv   PUNCH ~/scr/$JOB.dat
+setenv   PUNCH $SCR/$JOB.dat
 setenv  AOINTS $SCR/$JOB.F08
 setenv  MOINTS $SCR/$JOB.F09
 setenv DICTNRY $SCR/$JOB.F10
@@ -124,8 +124,8 @@
 setenv  SOINTY $SCR/$JOB.F32
 setenv  SOINTZ $SCR/$JOB.F33
 setenv  SORESC $SCR/$JOB.F34
-setenv   SIMEN ~/scr/$JOB.simen
-setenv  SIMCOR ~/scr/$JOB.simcor
+setenv   SIMEN $SCR/$JOB.simen
+setenv  SIMCOR $SCR/$JOB.simcor
 setenv GCILIST $SCR/$JOB.F37
 setenv HESSIAN $SCR/$JOB.F38
 setenv SOCCDAT $SCR/$JOB.F40
@@ -266,7 +266,7 @@
    if ($os == HP-UX)   set GMSPATH=/zr/mike/gamess
    if ($os == IRIX)    set GMSPATH=/home/hbar4/people/schmidt/gamess
    if ($os == IRIX64)  set GMSPATH=/home/hbar4/people/schmidt/gamess
-   if ($os == Linux)   set GMSPATH=/cu/mike/gamess
+   if ($os == Linux)   set GMSPATH=/usr/bin
    if ($os == OSF1)    set GMSPATH=/au/mike/gamess
    if ($os == SunOS)   set GMSPATH=/hf/mike/gamess
 #       special compilation for my old Sun (uname also replies SunOS)
